versions.tf

```diff
@@ -11,5 +12,9 @@ terraform {
       version = "~> 4.15.1"
       #Will allow installation of 4.15.1 and 4.15.10 but not 4.16.0
     }
+    docker = {
```

Comment:
Since we leveraged the docker provider in order to implement the `source_hash_code` solution, we must include the provider in our terraform configuration.